
This patch is intended for ApacheBench version 2.3.1, which is distributed
with Apache 2.2.10.  For some reason, millisecond resolution was removed
from the tsv output format.  This puts it back.


--- ab.c	2008-11-17 08:42:58.000000000 -0800
+++ ab.c-milliseconds	2008-11-17 08:42:47.000000000 -0800
@@ -983,7 +983,7 @@
                 fprintf(out, "%s\t%" APR_TIME_T_FMT "\t%" APR_TIME_T_FMT
                                "\t%" APR_TIME_T_FMT "\t%" APR_TIME_T_FMT
                                "\t%" APR_TIME_T_FMT "\n", tmstring,
-                        apr_time_sec(stats[i].starttime),
+                        stats[i].starttime,
                         ap_round_ms(stats[i].ctime),
                         ap_round_ms(stats[i].time - stats[i].ctime),
                         ap_round_ms(stats[i].time),
